src/index.css

```
@@ -1,2129 +1,226 @@
+  --ink:#0E1418;
+  --line:#22323A;
+  --line-soft:#1A272D;
+  --text:#DDE5E8;
+  --muted:#7D919B;
+  --dim:#56686F;
+  --faint:#3E5058;
+  --signal:#E8A33D;
+  --cool:#5FB8B0;
+  --rail:190px;
+  --pad:clamp(20px,5vw,72px);
+  --display:"Bricolage Grotesque","Helvetica Neue",Arial,sans-serif;
```

Stylelint (17.14.0)

[error] Expected "Arial" to be "arial" (value-keyword-case)

(value-keyword-case)